nss-evening-cohort-14/michelin-star-restaurant-dashboard-michelin-star-restaurant-dashboard-1
# UPDATE milestone complete

## Description
Completed the Update milestone. When authenticated and logged in, user can update items for staff, reservations, ingredients and menu. The items update patch the data in Firebase and the DOM is dynamically rendered after each item is updated.

## Link to deployed app
https://development--michelin-star-restaurant.netlify.app/#

## Related Issue
#25 
#27 
#28 
#30 

## Motivation and Context
The user can update the data of an existing item.

## How Can This Be Tested?
visit the deployed link, login, and click on the edit button on any item. A form will appear to update the values for that item. Click update and the dom will show the updated item.

## Screenshots (if appropriate):

## Types of changes
